tools/structprotogen/proto/proto.go

```diff
@@ -468,15 +468,13 @@ func getProtoBasicName(typ string) string {
 	switch typ {
 	case "bool":
 		return "bool"
-	case "int8", "int16":
-		return "fixed32"
-	case "int32":
+	case "int8", "int16", "int32":
```

Comment:
note: this is a big issue in `.proto` generation from resources

but this way it matches what `protoenc` does

Reply:
fix in https://github.com/siderolabs/talos/pull/11979